src/vivarium_nih_us_cvd/constants/results.py

```diff
@@ -28,13 +28,15 @@
 
 THROWAWAY_COLUMNS = [f'{state}_event_count' for state in models.STATES]
 
+# SDB: these don't match the data column names. Why arent the templates centralized?
+# FIXME [MIC-3230]: Update to match template
 TOTAL_POPULATION_COLUMN_TEMPLATE = 'total_population_{POP_STATE}'
```

Comment:
You'll end up changing these based on your observers.  Maybe best not to pre-optimize? Oftentimes, I don't edit these until I see exactly the columns the observers spat out into the results `output.hdf`.